plugins/importexport/doaj/classes/PubObjectCache.inc.php

```diff
@@ -0,0 +1,147 @@
+<?php
```

Comment:
Are you sure this got added as a symbolic link? OTOH this means a copy was added.